schemaregistry/schemaregistry.md

```diff
@@ -405,3 +497,196 @@ The returned ´Content-Type´ is the same that was passed when the schema versio
 was registered.
 
 The HEAD method SHOULD also be implemented.
+
+#### 3.4.3. Get a specific schema version by schema version URI
+
+Each schema version has a URI, as defined in [2.2.2](#222-schema-version-uri).
+
+This operation is a GET on `/schema?uri={uri}`, with the required parameter
```

Comment:
I'm probably missing something, but why can't they just use the URI instead of doing it indirectly via the "uri" query string?